emle/_backends/_deepmd.py

```diff
@@ -102,7 +102,7 @@ def __init__(self, model, deviation=None, deviation_threshold=None):
 
         self._max_f_std = None
 
-    def calculate(self, atomic_numbers, xyz, forces=True):
+    def calculate(self, atomic_numbers, xyz, forces=True, box=None):
```

Comment:
If you think we'll end up needing the `box` parameter more generally, then I'd be happy for this kwarg to be added t the base class method of the same name. I guess this involves more work updating the method signature and docstrings elsewhere, though.